### PR TITLE
 [Feat] Daily plan | Compare estimated UI Only

### PR DESCRIPTION
**Pull request by Innocent-Akim**
[Feature] 'Daily Plan' compare Estimated tasks time and Planned work hours #2614

Screenshot 2024-07-20 at 10 21 42 AM

### CODE REVIEW

File: apps/web/lib/features/daily-plan/daily-plan-compare-estimate-modal.tsx [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+                    memberInfo={memberInfo}
+                    edition={taskEdition}
+                />
+            </div>
+        </div>
+    );
+}
+
+
+export function ButtonAction({ closeModal, onClick }: { closeModal?: () => void, onClick?: () => void }) {
```
Comment (Cedric921) [inline apps/web/lib/features/daily-plan/daily-plan-compare-estimate-modal.tsx:86]: we can remane this component to `DailyPlanCompareActionButton`

File: apps/web/lib/features/daily-plan/daily-plan-compare-estimate-modal.tsx [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+            </Button>
+            <Button onClick={onClick} className='font-normal rounded-sm text-md h-9'>
+                Start working
+            </Button>
+        </div>
+    )
+}
+
+
+export function HeadTitle() {
```
Comment (Cedric921) [inline apps/web/lib/features/daily-plan/daily-plan-compare-estimate-modal.tsx:103]: We can also rename this component to `DailyPlanCompareHeader`

File: apps/web/lib/features/daily-plan/daily-plan-compare-estimate-modal.tsx [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+                    <Text.Heading as='h4' className='mb-3 text-center text-red-600'>*
+                    </Text.Heading>
+                </div>
+            </div>
+        </>
+    )
+}
+
+
+export function InputTime({ onChange }: { onChange?: (_: any) => void }) {
```
Comment (Cedric921) [inline apps/web/lib/features/daily-plan/daily-plan-compare-estimate-modal.tsx:125]: We can also rename to `DailyPlanWorkTimeInput`

File: apps/web/lib/features/daily-plan/daily-plan-compare-estimate-modal.tsx [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+                            <span>Please correct planned work hours or re-estimate task(s)</span>
+                        </div>
+                        <ButtonAction closeModal={closeModal} />
+                    </div>
+                </Card>
+            </div>
+        </Modal>
+    )
+}
+export function CardDailyPlan({ task, profile }: { task?: ITeamTask, profile: any }) {
```
Comment (Cedric921) [inline apps/web/lib/features/daily-plan/daily-plan-compare-estimate-modal.tsx:54]: let's rename to `DailyPlanTask`

**Comment by evereq** [comment]
@Cedric921 we should wait till @Innocent-Akim fix all those things from your comments? If so, maybe convert it to Draft for now?

**Comment by Innocent-Akim** [comment]
i'll change that in Next PR @Cedric921  @evereq

**Pull request merged**